src/interfaces/store-interfaces/state-interfaces/businessInformation-interface.ts

```diff
@@ -4,4 +4,5 @@ export interface BusinessInformationIF {
   identifier: string
   legalName?: string
   foundingDateTime?: string
+  hasRestrictions?: boolean
```

Comment:
It has been added to json but not to schema.

Reply:
Is it always optional, or is it only optional for now since it's not yet in the schema?

And does the API always return this property?

Reply:
The API always return but as we don't have it in the schema I set it as optional.